main/src/test/scala/org/clulab/numeric/TestNumericEntityRecognition.scala

```diff
@@ -175,8 +175,14 @@ class TestNumericEntityRecognition extends FlatSpec with Matchers {
   }
 
   it should "recognize \"month of X\" patterns" in {
-    ensure(sentence= "It was the month of January", Interval(3, 6), goldEntity= "DATE", goldNorm= "XXXX-01-XX")
-    ensure(sentence= "It was the month of January, 2020", Interval(3, 8), goldEntity= "DATE", goldNorm= "2020-01-XX")
+    ensure(sentence = "It was the month of January", Interval(3, 6), goldEntity = "DATE", goldNorm = "XXXX-01-XX")
+    ensure(sentence = "It was the month of January, 2020", Interval(3, 8), goldEntity = "DATE", goldNorm = "2020-01-XX")
+  }
+
+  it should "recognize date from holiday" in {
+    ensure(sentence= "Christmas 2016", Interval(0, 2), goldEntity= "DATE", goldNorm= "2016-12-25")
+    ensure(sentence= "Independence day", Interval(0, 2), goldEntity= "DATE", goldNorm= "XXXX-07-04")
+    ensure(sentence= "between Christmas and New Year", Interval(0, 4), goldEntity= "DATE-RANGE", goldNorm= "XXXX-12-25 -- XXXX-01-01")
```

Comment:
Could a test with an apostrophe be added, if only to document how it works or doesn't.  New Year's might suffice.

Reply:
I should have tested this in more detail, sorry. I will add more tests.